app/controllers/ErrorsReportedController.scala

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers
+
+import controllers.actions.{AuthAction, VerifiedEmailAction}
+import play.api.Logging
+import play.api.i18n.I18nSupport
+import play.api.mvc.{Action, AnyContent, MessagesControllerComponents}
+import uk.gov.hmrc.play.bootstrap.frontend.controller.FrontendController
+import views.html.errors_reported
+
+import javax.inject.{Inject, Singleton}
+import scala.concurrent.Future
+
+@Singleton
+class ErrorsReportedController @Inject() (
+  authenticate: AuthAction,
+  verifyEmail: VerifiedEmailAction,
+  mcc: MessagesControllerComponents,
+  rejectedNotificationPage: errors_reported
+) extends FrontendController(mcc) with I18nSupport with Logging {
+
+  def displayPage(id: String): Action[AnyContent] = (authenticate andThen verifyEmail).async { implicit request =>
+    val messages = messagesApi.preferred(request).messages
+    Future.successful(Ok(rejectedNotificationPage()(request, messages)))
+  }
+
+  def displayPageOnUnacceptedAmendment(actionId: String, draftDeclarationId: Option[String] = None): Action[AnyContent] =
+    (authenticate andThen verifyEmail).async { implicit request =>
+      val messages = messagesApi.preferred(request).messages
+      Future.successful(Ok(rejectedNotificationPage()(request, messages)))
+    }
+}
```

Comment:
Why do you need to define `messages`?
This controller could simply be reduced to...
```
  val displayPage: Action[AnyContent] = (authenticate andThen verifyEmail).async { implicit request =>
    Future.successful(Ok(rejectedNotificationPage()))
  }

  val displayPageOnUnacceptedAmendment: Action[AnyContent] = (authenticate andThen verifyEmail).async { implicit request =>
    Future.successful(Ok(rejectedNotificationPage()))
  }
```

Reply:
Removed :+1: